backend/internal/email_sender.go

```diff
@@ -29,9 +32,21 @@ var notifyPaymentRecordsMail []byte
 //go:embed templates/system_announcement.html
 var systemAnnouncementMail []byte
 
+//go:embed templates/notifications/*.html
+var emailTplFS embed.FS
+
+var emailTpls *template.Template
+
+var typeToTemplate = map[models.NotificationType]string{
```

Comment:
Why not implement a TemplateNamer interface that has TemplateName method on each type instead?